examples/benchmarks/nbody/java/nbody.java

```diff
@@ -0,0 +1,175 @@
+/* The Computer Language Benchmarks Game
```

Comment:
Maybe from an earlier version? Compare to https://github.com/softdevteam/warmup_experiment/blob/master/benchmarks/nbody/java/nbody.java and it looks different.